quizzes/midterm.md

```diff
@@ -177,12 +283,33 @@
 	  #no thanks
 	else
 	  #eat
-	```
+```
+
+```lang-ruby
+	case fruit
+		when 'apple'
+			#eat
+		when 'pear'
+			#eat
+		when durian
+			#no thanks
+		else
+			#eat
+	end
+```
+	
+28:  **T**  /  F Arrays can only hold `String` and `Hash` values?
+
+29:  What is the scope resolution operator? 
+
+```
 	
-1.  T  /  F Arrays can only hold `String` and `Hash` values?
+```
 
-1.  What is the scope resolution operator? 
+30: T / **F** Ruby expects that you might pass a hash as the last argument to a method.
```

Comment:
0/10 

Answer
True, If you are passing a hash as the last argument you can forego the brackets surrounding the hash.